packages/at_commons/lib/src/verb/operation_enum.dart

```diff
@@ -12,7 +12,7 @@ enum StrategyEnum { all, latest }
 String getStrategy(StrategyEnum? strategyEnum) =>
     '$strategyEnum'.split('.').last;
 
-enum MessageTypeEnum { key, text }
+enum MessageTypeEnum { key, @Deprecated('text based notification are deprecated') text }
```

Comment:
text based notifications

Reply:
addressed in [d577683](https://github.com/atsign-foundation/at_libraries/pull/571/commits/d577683e23362c3dc0c549d7123a8923008e85a3)